default.py

```diff
@@ -102,7 +101,7 @@ def hide_progress(self):
         """
         self.silent = True
 
-    def clean(self, video_type):
+    def clean(self, video_type, **kwargs ):
```

Comment:
I would like to have the header be ``def clean(self, video_type, expired_videos=None)`` as explained in the main comment thread. This saves having to add the ``kwargs.get('expired_videos', None)`` line and makes the allowed arguments look more clear to callers of the function. However, as I also explained, I would like to have a separate function for cleaning single videos as triggered by the monitor.